app/services/dialog_import_service.rb

```diff
@@ -113,6 +119,16 @@ def import_from_dialogs(dialogs)
           "resource_actions" => build_resource_actions(dialog)
         )
       )
+      associations.each do |a|
```

Comment:
I'm not so sure single letter variables are a good idea here. I understand it can be obvious `a` is associations, but `g` doesn't make much sense when you're doing `a.values`, and then `v` afterwards on `g.each` seems weird too. I'm pretty sure it means `groups` and `values`, but why not just use that and then it's easier to see at a glance exactly what we're talking about?